### PR TITLE
Change Source to Source0 and add sysmted for suse in mrsh

### PR DESCRIPTION
**Pull request by Yikun**
Signed-off-by: Yikun Jiang <yikunkero@gmail.com>

### CODE REVIEW

**Comment by Yikun** [comment]
cc @adrianreber

File: components/admin/mrsh/SPECS/mrsh.spec [hunk 1]
```diff
@@ -16,21 +16,22 @@
...
+Patch1:  mrsh-pam-suse.patch
+Patch2:  mrlogin-Don-t-use-union-wait.patch
+Patch3:  Add-force-to-libtoolize.patch
 BuildRequires: ncurses-devel pam-devel munge-devel make
 Requires: munge
 Provides: mrsh
 BuildRequires:  systemd-devel
 
 # support re-run of autogen
+BuildRequires: make
```
Comment (adrianreber) [inline components/admin/mrsh/SPECS/mrsh.spec]: make is already a BR in line 28

**Comment by adrianreber** [comment]
Please add following fix to the mrsh.spec file:
```diff
diff --git a/components/admin/mrsh/SPECS/mrsh.spec b/components/admin/mrsh/SPECS/mrsh.spec
index 082065c72..4071c1bfd 100644
--- a/components/admin/mrsh/SPECS/mrsh.spec
+++ b/components/admin/mrsh/SPECS/mrsh.spec
@@ -29,6 +29,9 @@ BuildRequires: ncurses-devel pam-devel munge-devel make
 Requires: munge
 Provides: mrsh
 BuildRequires:  systemd-devel
+%if 0%{?sles_version} || 0%{?suse_version}
+BuildRequires: systemd
+%endif
 
 # support re-run of autogen
 BuildRequires: autoconf
```

**Comment by Yikun** [comment]
@adrianreber Thanks!

**Comment by adrianreber** [comment]
Please squash the two commits into one.

**Pull request merged**